torchvision/datasets/__init__.py

```diff
@@ -128,3 +128,12 @@
     "InStereo2k",
     "ETH3DStereo",
 )
+
+
+def __getattr__(name):
```

Comment:
I'd suggest linking to the SLEP and explain why we need this (i.e. to avoid raising a warning when importing other stuff from `torchvision.datasets`